### ISSUE TITLE
Consider adding histogram support

### ISSUE THREAD

**Issue by szaydel** [marianogappa/chart#22]
This is a great tool to be sure, and I think it would be much better if it supported histograms. Often, when you are just trying to get a sense for some data, histogram is the most useful initial visualization.

**Comment by marianogappa** [comment]
I agree. I'd be reluctant to reinvent the wheel, but it seems the wheel hasn't been invented yet: https://github.com/chartjs/Chart.js/issues/3900

There doesn't seem to be Histogram support planned in the short term. I haven't ever coded a histogram chart before, so this sounds like an interesting challenge. Let me think about it.

PRs are also welcome, of course.

**Comment by marianogappa** [comment]
Some slow progress seems to be happening on the chartjs side:
https://github.com/chartjs/Chart.js/issues/3900

I'm sorry but I've been too busy to work on this so far; and I don't think I'll be able to any time soon.

**Comment by szaydel** [comment]
It makes sense to wait for the library to implement support for it. If I have spare time, I may just try to implement something with bars, but not really sure if extending from Chart.js into this app is most sensible thing to do.

**Comment by spitzbubchen** [comment]
I'm also looking for histogram support for scientific plotting purposes. I'm wonder if someone could post a jsfiddle using some rough workaround approach (bar charts or something) until this feature is supported.

**Comment by spitzbubchen** [comment]
Here is a jsfiddle that I cobbled together over lunch using a bar chart to make a histogram.
